nautobot/docs/guides/graphql.md

```diff
@@ -0,0 +1,1041 @@
+      },
+      {
+        "name": "lax-leaf-07"
+      },
+      {
+        "name": "lax-leaf-08"
+      },
+      {
+        "name": "lax-leaf-09"
+      },
+      {
+        "name": "lax-leaf-10"
+      },
+      {
+        "name": "lhr-edge-01"
+      },
+      {
+        "name": "lhr-edge-02"
+      },
+      {
+        "name": "lhr-leaf-01"
+      },
+      {
+        "name": "lhr-leaf-02"
+      },
+      {
+        "name": "lhr-leaf-03"
+      },
+      {
+        "name": "lhr-leaf-04"
+      },
+      {
+        "name": "lhr-leaf-05"
+      },
+      {
+        "name": "lhr-leaf-06"
+      },
+      {
+        "name": "lhr-leaf-07"
+      },
+      {
+        "name": "lhr-leaf-08"
+      },
+      {
+        "name": "nyc-bb-01.infra.ntc.com"
+      },
+      {
+        "name": "nyc-leaf-01.infra.ntc.com"
+      },
+      {
+        "name": "nyc-leaf-02.infra.ntc.com"
+      },
+      {
+        "name": "nyc-rtr-01.infra.ntc.com"
+      },
+      {
+        "name": "nyc-rtr-02.infra.ntc.com"
+      },
+      {
+        "name": "nyc-spine-01.infra.ntc.com"
+      },
+      {
+        "name": "nyc-spine-02.infra.ntc.com"
+      },
+      {
+        "name": "ord-edge-01"
+      },
+      {
+        "name": "ord-edge-02"
+      },
+      {
+        "name": "ord-leaf-01"
+      },
+      {
+        "name": "ord-leaf-02"
+      },
+      {
+        "name": "ord-leaf-03"
+      },
+      {
+        "name": "ord-leaf-04"
+      },
+      {
+        "name": "ord-leaf-05"
+      },
+      {
+        "name": "ord-leaf-06"
+      },
+      {
+        "name": "ord-leaf-07"
+      },
+      {
+        "name": "ord-leaf-08"
+      },
+      {
+        "name": "pek-edge-01"
+      },
+      {
+        "name": "pek-edge-02"
+      },
+      {
+        "name": "pek-leaf-01"
+      },
+      {
+        "name": "pek-leaf-02"
+      },
+      {
+        "name": "pek-leaf-03"
+      },
+      {
+        "name": "pek-leaf-04"
+      },
+      {
+        "name": "pek-leaf-05"
+      },
+      {
+        "name": "pek-leaf-06"
+      },
+      {
+        "name": "pek-leaf-07"
+      },
+      {
+        "name": "pek-leaf-08"
+      },
+      {
+        "name": "pvg-edge-01"
+      },
+      {
+        "name": "pvg-edge-02"
+      },
+      {
+        "name": "pvg-leaf-01"
+      },
+      {
+        "name": "pvg-leaf-02"
+      },
+      {
+        "name": "pvg-leaf-03"
+      },
+      {
+        "name": "pvg-leaf-04"
+      },
+      {
+        "name": "sin-edge-01"
+      },
+      {
+        "name": "sin-edge-02"
+      },
+      {
+        "name": "sin-leaf-01"
+      },
+      {
+        "name": "sin-leaf-02"
+      },
+      {
+        "name": "sin-leaf-03"
+      },
+      {
+        "name": "sin-leaf-04"
+      },
+      {
+        "name": "sin-leaf-05"
+      },
+      {
+        "name": "sin-leaf-06"
+      },
+      {
+        "name": "sin-leaf-07"
+      },
+      {
+        "name": "sin-leaf-08"
+      }
+    ]
+  }
+}
+```
```

Comment:
This is way overkill as example output. :-) Maybe cut it down to a half-dozen or so entries?

Reply:
Removed screen shots of the output and updated the output per @dgjustice suggestion.